generic/tests/cfg/boot_vm_in_hugepage.cfg

```diff
@@ -10,8 +10,8 @@
     pre_command_noncritical = yes
     pre_command = "echo 3 > /proc/sys/vm/drop_caches"
     variants:
-        - 2M_hugepage:
-            expected_hugepage_size = 2048
+        - @default:
```

Comment:
No, there is a check in the test_setup.py, please check it.
and there is hugepage check string for different hugepage size.